src/test/java/com/google/javascript/clutz/multifilePartial/default_object_exporter.js

```diff
@@ -0,0 +1,10 @@
+goog.module('default.object.exporter');
+
+/**
+ * @constructor
+ */
+function BaseClass() {
+
+}
+
+exports = {BaseClass};
```

Comment:
Just `exports.BaseClass = BaseClass`, I think that's more common.

Reply:
Rewrote it to look more like the output from tsickle.